app/views/content/international-returners.md

```diff
@@ -61,16 +61,15 @@ right_column:
       icon: "icon-calendar"
 ---
 
-Are you a UK-trained teacher and UK citizen currently working abroad? Our
+Are you a UK-trained teacher and UK citizen currently teaching abroad? Our
 children and young people need your international skills, experience and
 perspective.
 
 If you're thinking about coming home to teach, we want to help.
 
 If you are a non-UK citizen interested in teaching or training to teach in
-England, please read our
-[guidance for international teachers and trainees](/train-to-teach-in-england-as-an-international-student) and [register for our next online event](/event-categories/online-q-as)
-to learn how we can support your journey to the UK.
+England, please read our [guidance for international teachers](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk) or
+[guidance for international trainees](/train-to-teach-in-england-as-an-international-student). 
 
 ### Return to teaching success stories
```

Comment:
we shouldn't really be jumping straight from h1s to h3s - this is an accessibility issue, not great for screen readers

Reply:
Thank you, have taken up